src/main/webapp/resources/js/pages/admin/components/Admin.jsx

```diff
@@ -58,30 +64,46 @@ export default function Admin() {
         <AdminHeader />
         <Content>
           <Suspense fallback={<ContentLoading />}>
-            <Router>
-              <BasicStats path={`${DEFAULT_URL}/${ADMIN.STATISTICS}`} default />
-              <AdvancedStatistics
-                path={`${DEFAULT_URL}/${ADMIN.STATISTICS}/:statType`}
-              />
-              <AdminUsersPage path={`${DEFAULT_URL}/${ADMIN.USERS}`} />
-              <AdminUserGroupsPage
-                path={`${DEFAULT_URL}/${ADMIN.USERGROUPS}/*`}
-              />
-              <ClientListingPage path={`${DEFAULT_URL}/${ADMIN.CLIENTS}`} />
-              <AdminRemoteApiPage path={`${DEFAULT_URL}/${ADMIN.REMOTEAPI}`} />
-              <AdminRemoteApiDetailsPage
-                path={`${DEFAULT_URL}/${ADMIN.REMOTEAPI}/:remoteId`}
-              />
-              <AdminSequencingRunsPage
-                path={`${DEFAULT_URL}/${ADMIN.SEQUENCINGRUNS}`}
-              />
-              <AdminNcbiExportsPage
-                path={`${DEFAULT_URL}/${ADMIN.NCBIEXPORTS}`}
-              />
-              <AnnouncementAdminPage
-                path={`${DEFAULT_URL}/${ADMIN.ANNOUNCEMENTS}`}
-              />
-            </Router>
+            <Routes>
+              <Route path={DEFAULT_URL} element={<AdminContent />}>
```

Comment:
When going to the route `http://localhost:8080/admin` the `Statistics` link is highlighted but the page is not loaded into the div

Reply:
Looks like they changed that up a bit.  Updated in 12ceb36

Reply:
Thanks for fixing that part of it. But now another issue has appeared.

So I was just trying out the react router admin panel stuff. If you go into one of the statistics charts and then click the back button the statistics page doesn't load. Just a blank div is displayed
Might have something to do with the url:
http://localhost:8080/admin/statistics causes the statistics not to display however, http://localhost:8080/admin loads the statistics page

Reply:
Thanks for fixing this!